utils/nctl/sh/assets/upgrade_from_stage_single_node.sh

```diff
@@ -253,36 +253,96 @@ function _setup_asset_node_config_workaround_1()
     python3 -c "${SCRIPT[*]}"
 }
 
+function _validators_string()
```

Comment:
It doesn't seem that we use this function. Can it be removed?

Reply:
It has been removed in https://github.com/casper-network/casper-node/pull/3451/commits/7e6ad1cf023213409a81fbce841f8fb43d5ca6e1